scripts/exgfs_wave_prdgen_gridded.sh

```diff
@@ -206,7 +209,8 @@ grids=${GEMPAK_GRIDS:-ak_10m at_10m ep_10m wc_10m glo_30m}
        echo "${msg}"
        #set_trace
        echo "${RUN} wave prdgen ${date} ${cycle} : error in tocgrib2."
-       err=5;export err;err_chk
+       export err=5
+       err_chk
```

Comment:
Another area to compress...move fatal error message to `err_chk` line and delete other echos and such in lines 202 to 211.